content/02.main-text.md

```diff
@@ -7,7 +7,7 @@ Open research – which includes sharing code, data, and manuscripts – benefit
 Here we describe the benefits of writing review articles openly, where the planning, organizing, writing, and editing occur collaboratively in a public forum where participants are free to join as they wish.
 Reviews presenting the state of the art in a scientific field are often prepared by a single research group or a small team of colleagues.
 In contrast, broadly opening the writing process to anyone engaged in the topic can help maximize the review's value by facilitating the representation of diverse opinions and the broad coverage of relevant research.
-Review authors can engage with the authors of original research to clarify their methods and results and present them accurately (for example, [@url:https://github.com/greenelab/deep-review/issues/213]).
+Review authors can engage with the authors of original research to clarify their methods and results and present them accurately (for example, @url:https://github.com/greenelab/deep-review/issues/213).
```

Comment:
> How in-text citations are rendered depends on the selected CSL style, I believe.

Got it.  That was the part I wasn't completely sure about.

Thanks for taking care of the merge.